package.json

```diff
@@ -25,11 +25,11 @@
     "package-maven-build-scan-capture-extension": "mvn clean package -f maven-build-scan-capture-extension/pom.xml",
     "package-build-scan-setup-maven-main": "ncc build build-scan-setup-maven/src/main.ts --out build-scan-setup-maven/dist/main",
     "package-build-scan-setup-maven-post": "ncc build build-scan-setup-maven/src/post.ts --out build-scan-setup-maven/dist/post",
-    "package-build-scan-setup-maven": "npm run package-build-scan-setup-maven-main && npm run package-build-scan-setup-maven-post && npm run package-maven-build-scan-capture-extension",
+    "package-build-scan-setup-maven": "npm-run-all --parallel package-build-scan-setup-maven-main package-build-scan-setup-maven-post package-maven-build-scan-capture-extension",
     "package-build-scan-setup": "npm run package-build-scan-setup-maven",
     "package-build-scan-publish-maven": "ncc build build-scan-publish-maven/src/main.ts --out build-scan-publish-maven/dist",
     "package-build-scan-publish": "npm run package-build-scan-publish-maven",
-    "package-build-scan": "npm run package-build-scan-setup && npm run package-build-scan-publish",
+    "package-build-scan": "npm-run-all --parallel package-build-scan-setup package-build-scan-publish",
```

Comment:
The `npm-run-all` tool accepts wildcard arguments, so you can probably reorganize this to be simpler.
Something like:
```
"package": "npm-run-all --parallel package-*"
```

I also find it useful to have a `npm run build` command that runs `format:write`, `lint` and `package`, but doesn't run any tests.

Reply:
I applied your suggestion 👍